drain/test_drain.py

```diff
@@ -0,0 +1,42 @@
+from utils import wait_for
+from pyln.testing.fixtures import *
+
+pluginopt = {'plugin': os.path.join(os.path.dirname(__file__), "drain.py")}
+
+def test_drain(node_factory):
+    # build a simple circular setup to run tests on
+    # l1, l2, l3, l4 = node_factory.line_graph(4, announce_channels=True, wait_for_announce=True, opts=pluginopt)
+    # ValueError: ('Error waiting for {}', <function NodeFactory.line_graph.<locals>.<lambda> at 0x7f299deb67b8>)
+    l1, l2, l3, l4 = node_factory.line_graph(4, announce_channels=True, opts=pluginopt)
```

Comment:
This might require the `@unittest.skipIf(not DEVELOPER)` annotation on the test since otherwise the gossip sync is too slow ^^